src/components/Admin/ClientFieldDriver.js

```diff
@@ -1,10 +1,12 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import VerifyButton from './verification/VerifyButton';
 
 const ClientFieldDriver = ({
   driver: {
     id, phoneNumber, firstName, lastName, isActive, isVerifiedByAdmin,
   },
+  onClick,
 }) => {
   let verified = 'bg-danger';
```

Comment:
rename verified variable name to `verifiedClassName`
`const verifiedClassName = isVerifiedByAdmin ? 'bg-success' : 'bg-danger';
`const activeClassName = isActive && isVerifiedByAdmin ? 'bg-success' : 'bg-danger';